csm_test_utils/autoscaling.py

```diff
@@ -0,0 +1,49 @@
+"""Get cloud eye notifications and check that autoscaling is working"""
+import json
+
+import requests
+from flask import Flask, request
+from influx_line_protocol import Metric, MetricCollection
+
+from .common import base_parser, sub_parsers
+
+app = Flask(__name__)
+
+AS_RESULT = "as_result"
+
+
+@app.route("/smn", methods=["POST"])
+def smn():
+    if request.method == "POST":
+        response = request.get_json()
+        if "subscribe_url" in response:
+            requests.get(response["subscribe_url"])
+        else:
+            report(json.loads(response["message"]))
+        return response
+
```

Comment:
Pylint: `R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)`